content/90.back-matter.md

```diff
@@ -3,56 +3,133 @@
+Token association of PC1 shows the separation of cell biology and informatics-related fields through tokens: "empirical", "estimates" and "statistics" depicted in orange and "cultured" and "overexpressing" shown in blue (Supplementary Figure {@fig:topic_analysis_panels}B and Supplementary Table {@tbl:pc1_loadings_table}).
+Association for PC2 shows the separation of bioinformatics and neuroscience via tokens: "genomic", "genome" and "genomes" depicted in orange and "evoked", "stimulus" and "stimulation" shown in blue (Supplementary  Figure {@fig:topic_analysis_panels}C and Supplementary Table {@tbl:pc2_loadings_table}).
+
+Examining the value for PC1 across all author-selected categories revealed an ordering of fields from cell biology to informatics-related disciplines (Figure {@fig:topic_analysis_panels}D).
+These results suggest that a primary driver of the variability within the language used in bioRxiv could be the divide between informatics and cell biology approaches.
+A similar analysis for PC2 suggested that neuroscience and bioinformatics present a similar language continuum (Supplementary Figure {@fig:topic_analysis_panels}E).
+This result supports the notion that bioRxiv contains an influx of neuroscience and bioinformatics-related research results. 
+For both of the top two PCs, the submitter-selected category of systems biology preprints was near the middle of the distribution and had a relatively large interquartile range when compared with other categories (Supplementary Figure {@fig:topic_analysis_panels}D and {@fig:topic_analysis_panels}E), suggesting that systems biology is a broader subfield containing both informatics and cellular biology approaches. 
+
+Examining the top five highest-scoring and bottom five lowest-scoring systems biology preprints along PC1 reinforces its dichotomous theme (Supplementary Table {@tbl:five_pc1_table}).
+Preprints with the highest values [@doi:10.1101/197400;@doi:10.1101/825943;@doi:10.1101/044818;@doi:10.1101/769299;@doi:10.1101/107250] included software packages, machine learning analyses, and other computational biology manuscripts, while preprints with the lowest values [@doi:10.1101/455048;@doi:10.1101/371922;@doi:10.1101/733162;@doi:10.1101/745943;@doi:10.1101/754572] were focused on cellular signaling and protein activity.
+We provide the rest of our 50 generated PCs in our online repository (see Software and Data Availability).
+
+| Title [citation]     | PC1  | License | Figure Thumbnail |
+|--------------------------------|-------------------|------------|----------|
+| Conditional Robust Calibration (CRC): a new computational Bayesian methodology for model parameters estimation and identifiability analysis [@doi:10.1101/197400] | 4.522818390064091 | None         | ![](images/paper-thumbnails/pc1/197400_thumbnail.png) | 
```

Comment:
I really thought this analysis was cool 😭 . Even if this gets sent to the supplement, we need to refer readers to the supplement somewhere. What do you think about adding a few sentences to the end of the first part of the results section that summarize this section and point folks here if they are interested?